src/PDS.API/Program.cs

```diff
@@ -81,11 +93,85 @@ private static async Task GetConsignments()
                 Console.WriteLine($"Text : {eventdata.Text}");
                 Console.WriteLine($"Text : {eventdata.CreatedAt}");
                 Console.WriteLine(" ");
-
             }
         }
+    }
+    private static async Task GetCustomerById()
+    {
+        Console.WriteLine("Please input the customer key");
+        string id = Console.ReadLine()?? "";
```

Comment:
Add whitespace between Console.ReadLine() and question marks. `string id = Console.ReadLine() ?? "";`